### ISSUE TITLE
Sign-out button on RStudio doesn't reroute to JupyterHub

### ISSUE THREAD

**Issue by cdbeon** [berkeley-dsep-infra/hubs#3134]
### Bug description

Previously, when clicking on the "Sign Out" button on RStudio, it would navigate to JupyerHub (where you can access the Home/Token/Admin tabs). Now, it just leads to a blank 'unauthorized' page.

### Environment & setup

- Hub: Public Health
- Language: R / JupyterHub

### How to reproduce

1. Navigate to `publichealth.datahub.berkeley.edu`
2. Click on the "Sign Out" button on the top-right of RStudio (next to the red "Quit the Current R Session" button)
3. A blank HTML page will pop up with "Unauthorized" on the top left.

**Comment by balajialg** [comment]
Thanks, @cdbeon for raising this request! Apparently, this behavior seems to extend to other hubs including R and Datahub.

@ryanlovett Is this a result of the RStudio version upgrade to 1.4? Should we raise this issue in their GitHub repository instead (if it exists)

**Comment by ryanlovett** [comment]
@balajialg I don't think it is an RStudio bug, just an issue with how it behaves with our proxying. I created https://github.com/jupyterhub/jupyter-rsession-proxy/issues/122 to track this.
